Scripts/pre-commit.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -eu
```

Comment:
Ok, but that's for you and no else no?  You leave here, but it's not that normal to add these utilities that you like to a shared repo